python/cog/json.py

```diff
@@ -26,7 +26,10 @@ def make_encodeable(obj: Any) -> Any:  # pylint: disable=too-many-return-stateme
     """
 
     if isinstance(obj, BaseModel):
-        return make_encodeable(obj.dict(exclude_unset=True))
+        if PYDANTIC_V2:
+            return make_encodeable(obj.model_dump(exclude_unset=True))
```

Comment:
What's the deal with this rather than `obj.dict` or `obj.json`?

Reply:
pydantic2 deprecates `obj.dict`